drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -1978,6 +1987,14 @@ void RasterizerSceneGLES3::render_scene(const Ref<RenderSceneBuffers> &p_render_
 		_draw_sky(render_data.environment, render_data.cam_projection, render_data.cam_transform, sky_energy_multiplier);
 	}
 
+	if (rt->copy_fbo > 0) {
+		glReadBuffer(GL_COLOR_ATTACHMENT0);
+		glBindFramebuffer(GL_DRAW_FRAMEBUFFER, rt->copy_fbo);
+		glBlitFramebuffer(0, 0, rt->size.x, rt->size.y,
```

Comment:
This needs multiview support, but we can add that in later. In theory it should be enough to set the z component to `view_count` if multiview is used